apps/explorer/src/app/routes/router-config.tsx

```diff
@@ -241,6 +241,37 @@ const validators: Route[] = FLAGS.EXPLORER_VALIDATORS
     ]
   : [];
 
+const oraclesRoutes: Route[] = FLAGS.EXPLORER_ORACLES
```

Comment:
hey, if it is a mainnet issue could we please merge it in `main` instead of develop.
I have recently introduced some big schema-breaking changes that will break on the current core version.